src/components/onramp/index.tsx

```diff
@@ -0,0 +1,39 @@
+import {
+  Modal,
+  ModalBody,
+  ModalCloseButton,
+  ModalContent,
+  ModalHeader,
+  ModalOverlay,
+} from '@chakra-ui/react'
+
+import { useOnramperUrl } from '@/components/onramp/utils'
+
+type OnrampModalProps = {
+  address?: string
+  isOpen: boolean
+  onClose: () => void
+}
+
+export const OnrampModal = (props: OnrampModalProps) => {
+  const src = useOnramperUrl()
+  const { isOpen, onClose } = props
+  return (
+    <Modal onClose={onClose} isOpen={isOpen} isCentered scrollBehavior='inside'>
+      <ModalOverlay className='bg-ic-black bg-opacity-60 backdrop-blur' />
+      <ModalContent className='border-ic-gray-100 bg-ic-white mx-0 my-0 rounded-xl border-2 p-0'>
```

Comment:
@0xonramp forgive me for using chakra for the modals but at least here I used tailwind. 😅  we can replace all of the modals together tho if we find a tailwind component. a ticket for this is already in the backlog.